### PR TITLE
clarify dbt commands 

### PR DESCRIPTION
**Pull request by mirnawong1**
this page has confused me since i started and wanted to turn it into a table so it's clearer what commands you can use in cloud and the cli (rather than inferred). this will change at a later point potentially

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

**Comment by mirnawong1** [comment]
will work on this when im back apr 13th

**Comment by owenprough-sift** [comment]
Should the commands be alphabetized in the table, or is there a reason they are not?

**Comment by mirnawong1** [comment]
hey @owenprough-sift - thanks for that great suggestion and I'm picking this pr back up this week. i think that's a great idea and I'll make sure i take your suggestion on ✨

**Comment by mirnawong1** [comment]
hey @matthewshaver - can i ask for your review for this pr? i think you're working on these pages so wanted to make sure there was no overlap? thank you!

File: website/docs/reference/dbt-commands.md [hunk 1]
```diff
@@ -6,15 +6,18 @@ dbt is typically run one of two ways:
...
+The following sections outline the commands supported by dbt and their relevant flags. Note that some commands are only supported when using the CLI. 
 
 For information about selecting models on the command line, consult the docs on [Model selection syntax](node-selection/syntax).
 
+### Available commands
+
+Select the tabs that are relevant to the your development workflow. For example, if you develop in the dbt Cloud IDE, select the "Available commands in dbt Cloud" tab. 
 
 <Tabs>
 <TabItem value="cloud" label="Available commands in dbt Cloud">
```
Comment (runleonarun) [inline website/docs/reference/dbt-commands.md]:
```suggestion
<TabItem value="cloud" label="dbt Cloud">
```

File: website/docs/reference/dbt-commands.md [hunk 2]
```diff
@@ -32,7 +35,7 @@ Use the following dbt commands in the [dbt Cloud IDE](/docs/cloud/develop-in-the
 
 <TabItem value="cli" label="Available commands in the CLI">
```
Comment (runleonarun) [inline website/docs/reference/dbt-commands.md]:
```suggestion
<TabItem value="cli" label="CLI">
```

**Review by runleonarun — COMMENTED** [review]
One tiny sugg to tighten up the tabs. Now that we have a section header the tabb titles are repetitive. 

@mirnawong1 I don't see a table as mentioned in the PR description? Did i miss something or did we change course? Otherwise, feel free to merge!

**Comment by mirnawong1** [comment]
Thanks so much for that @runleonarun ! Yes i originally used a table, which looked awful! So tried tabs and forgot to update. Apologies and great suggestions!

**Review by runleonarun — APPROVED** [review]
Looks good :shipit:

**Pull request merged**